.circleci/scripts/binary_checkout.sh

```diff
@@ -20,21 +20,8 @@ export BUILDER_ROOT="$workdir/builder"
 # Clone the Pytorch branch
 git clone https://github.com/pytorch/pytorch.git "$PYTORCH_ROOT"
 pushd "$PYTORCH_ROOT"
-if [[ -n "$CIRCLE_PR_NUMBER" ]]; then
-  # "smoke" binary build on PRs
-  git fetch --force origin "pull/${CIRCLE_PR_NUMBER}/head:remotes/origin/pull/${CIRCLE_PR_NUMBER}"
-  git reset --hard "$CIRCLE_SHA1"
-  git checkout -q -B "$CIRCLE_BRANCH"
-  git reset --hard "$CIRCLE_SHA1"
-elif [[ -n "$CIRCLE_SHA1" ]]; then
-  # "smoke" binary build on master on PR merges
-  git reset --hard "$CIRCLE_SHA1"
-  git checkout -q -B master
-else
-  # nightly binary builds. These run at 05:05 UTC every day. 
-  last_commit="$(git rev-list --before "$(date -u +%Y-%m-%d) 05:00" --max-count 1 HEAD)"
-  git checkout "$last_commit"
-fi
+git checkout my_branch
```

Comment:
Note that after https://github.com/pytorch/pytorch/pull/19674 lands, these changes should be included in my_branch itself.